build.sbt

```diff
@@ -250,7 +250,7 @@ lazy val docs = akkaModule("akka-docs")
       "extref.akka.http.base_url" -> "https://doc.akka.io/docs/akka-http/current/%s",
       "extref.wikipedia.base_url" -> "https://en.wikipedia.org/wiki/%s",
       "extref.github.base_url" -> (GitHub.url(version.value) + "/%s"), // for links to our sources
-      "extref.samples.base_url" -> "https://developer.lightbend.com/start/?group=akka&project=%s",
+      "extref.samples.base_url" -> "https://developer.lightbend.com/start/?group=akka&amp;project=%s",
```

Comment:
You double checked (or know for sure) that this works I assume?

Reply:
I mean that there isn't any further escaping or so

Reply:
yes, tested in firefox, no, doesn't seem to be any additional escaping elsewhere.